README.md

```diff
@@ -6,12 +6,12 @@ workloads. The dataplane-operator prepares the nodes with enough operating
 system configuration so that they are ready for hosting other required
 OpenStack services and workloads.
 
-See [contributing](contributing.md) for notes for developers and
+See [contributing](docs/contributing.md) for notes for developers and
```

Comment:
this change would break mkdocs
https://logserver.rdoproject.org/52/352/c703497978efd6bcb15b1c115eb5b99171dd52eb/github-check/dataplane-operator-docs-preview/588716e/docs/index.html

Reply:
:+1: We don't want to break mkdocs.

Reply:
Ack